lib/ansible/modules/systemd_service.py

```diff
@@ -26,7 +26,8 @@
     state:
         description:
             - V(started)/V(stopped) are idempotent actions that will not run commands unless necessary.
-              V(restarted) will always bounce the unit. V(reloaded) will always reload.
+              V(restarted) will always bounce the unit.
+              V(reloaded) will always reload and if the service is not running at the moment of the reload, it is restarted.
```

Comment:
it is started .. not really 're'

Reply:
Right, the service is only started if not running

Reply:
not what i meant, its not 're'started .. it is 'started'